SPECS/containerized-data-importer/containerized-data-importer.spec

```
@@ -113,7 +113,7 @@ tar --strip-components=1 -xf %{SOURCE0}
 export GOPATH=%{_builddir}/go
 export GOFLAGS+="-buildmode=pie -mod=vendor"
 env \
-CDI_SOURCE_DATE_EPOCH="$(date -r LICENSE +%s)" \
+CDI_SOURCE_DATE_EPOCH="$(date -r LICENSE +%{s})" \
```

This looks like a linting bug - please revert.
```suggestion
CDI_SOURCE_DATE_EPOCH="$(date -r LICENSE +%s)" \
```